contrib/linux/dosbox-staging.metainfo.xml

```diff
@@ -31,6 +31,11 @@
       DOSBox Staging is highly configurable and sufficiently-optimized to run
       any DOS game on a modern computer.
     </p>
+    <p>Relationship to DOSBox:</p>
+    <p>
+      DOSBox Staging is separate from and not supported by the SourceForge-hosted
+      DOSBox project or its development team, the DOSBox Team.
+    </p>
```

Comment:
Do we really need to advertise "we are separate from DOSBox in here as well?" We already do this:

- in our homepage and
- in "about" page
- in README.md file

It has no meaning to users - only to people interested in contributing, and those people will find it soon enough.

I would prefer to keep our description in metainfo briefer and omit this pointless info.

If Debian package description requires something more, then update only that description according to Debian maintainers' wishes.

Reply:
My thinking was, that when Debian called for clear mention that DOSBox Staging was not part of DOSBox in the long description that would include the Software Center since it is user facing, and perhaps to many users the only place they would see such a description.

But yeah, I'll just add it to debian/control.

Regards the icon, it will be fine once it gets onto official repos. If I inject DEP11 into my local repo the icon appears in the software centre like magic.

The only outstanding is about vs release notes, so I'll just close this Draft PR :smile: